Bottom: 80590dfcdfaeabe9f87e9843d5d4c3a8acc5a7c5
Top:    ef1b68ec6da26ba44ca69f215eb5e3d6b3d4f1f8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 23:18:12 +0000

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index 1ef631b..3bd66f6 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3093,7 +3093,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
